src/docs/product/releases/health/index.mdx

```diff
@@ -0,0 +1,62 @@
+---
+title: Health
+sidebar_order: 40
+redirect_from:
+  - /workflow/releases/health/
+  - /product/releases/health/
+description: "Monitor the health of releases by observing user adoption, usage of the application, percentage of crashes, and session data."
+---
+
+Monitor the health of releases by observing user adoption, usage of the application, percentage of [crashes](#crash), and [session data](#session-data). Release health will provide insight into the impact of crashes and bugs as it relates to user experience, and reveal trends with each new issue through the release details, graphs, and filters.
+
+## Enabling Release Health
+
+Many SDKs automatically manage the start and end of sessions when the SDK is initialized, but release health configuration is key to ensuring you are receiving data that is useful, so check our information on [which SDKs support release health](/product/releases/health/setup/) for links to configuring the SDK, as needed.
+
+## Data Filtering
+
+Sentry provides [inbound data filters](/product/error-monitoring/filtering/) to filter data into your project. Data filters are only partially supported for session events.
+
+## What Release Health Monitors
```

Comment:
May I ask, where did you get the term "monitor" from? We do have a concept of monitors in Sentry, but it does not relate to sessions at all.

Reply:
I pulled it from other content, but I agree we should avoid specifically-used terms such as monitor and metrics in this content.